dinghy-lib/src/errors.rs

```diff
@@ -15,15 +16,14 @@ error_chain! {
             description("Cannot compile selected packages for the selected platform")
             display("{:?} cannot be compiled for the selected platform (see project's [package.metadata.dinghy] in Cargo.toml)", packages)
         }
-        Cargo(err: ::cargo::CargoError) {
+        Cargo(err: String) {
```

Comment:
This could probably be renamed from `Cargo` because it's technically from `failure` but given that the `cargo` crate is the only thing passing up `failure` errors, it's arguably accurate.